app/src/main/java/com/internship/tvseries/data/repository/backend/BackendPopularRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.internship.tvseries.data.repository.backend;
+
+import com.internship.tvseries.data.model.Result;
+import com.internship.tvseries.data.repository.TvRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class BackendPopularRepository implements TvRepository {
```

Comment:
Let's rename this to LynxPopularRepository. "Backend" name was just a temporary idea. Please check other fields with this name in it and rename them